packages/cardpay-sdk/sdk/utils/general-utils.ts

```diff
@@ -48,11 +50,17 @@ export interface ErrorFragment {
   inputs: string[];
 }
 
+// We don't care if it's the patched version from the SDK or one instantiated directly from ethers.js
```

Comment:
We do care about it I think because unpatched JsonRpcProvider had a different way to handle a reverted message, it might cause some scheduler module functions doesn't work properly such as `estimateExecutionGas` and `createSafeEstimation`.